doc/2/plugins/plugin-context/constructors/request/index.md

```diff
@@ -58,7 +58,7 @@ The `options` argument accepts the following parameters:
 | -------------- | ----------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `connection`   | <pre>object</pre>                                           | <SinceBadge version="1.4.1" /> Connection information (see the <a href=https://github.com/kuzzleio/kuzzle-common-objects/blob/master/README.md#requestcontextconnection-object-format>connection</a> object documentation) |
 | `connectionId` | <pre>string</pre>                                           | <DeprecatedBadge version="1.4.1" /> Connection unique identifier                                                                                                                                                           |
-| `error`        | [`KuzzleError`](/core/1/plugins/errors), Error</pre> | Sets the request response with the provided error                                                                                                                                                                          |
+| `error`        | [`KuzzleError`](/core/2/plugins/errors), Error</pre> | Sets the request response with the provided error                                                                                                                                                                          |
```

Comment:
`[`KuzzleError`](/core/2/plugins/errors)` lead to a 404 and I don't think that the link should be in because the type it's hard to see

![image](https://user-images.githubusercontent.com/19224148/63166650-58f5a400-c02f-11e9-9a29-d815745448b4.png)